src/backend/common/DefaultMemoryManager.cpp

```diff
@@ -206,8 +208,8 @@ void *DefaultMemoryManager::alloc(bool user_lock, const unsigned ndims,
 size_t DefaultMemoryManager::allocated(void *ptr) {
     if (!ptr) return 0;
     memory_info &current = this->getCurrentMemoryInfo();
-    locked_iter iter     = current.locked_map.find((void *)ptr);
```

Comment:
This type alias should be removed from header
https://github.com/arrayfire/arrayfire/blob/3693e6a0ea5ec89a39f4699102db1967de3b923d/src/backend/common/DefaultMemoryManager.hpp#L39